en/08_Changelogs/5.4.0.md

```diff
@@ -32,6 +33,18 @@ SilverStripe\ORM\FieldType\DBPolymorphicForeignKey:
     Class: "DBClassNameVarchar('SilverStripe\\ORM\\DataObject', ['index' => false])"
 ```
 
+### Reports Quality of life updates
+
+Numerous slight adjustments have been made to the base ReportAdmin class for a better experience.
```

Comment:
```suggestion
Numerous slight adjustments have been made to the [`ReportAdmin`](api:SilverStripe\Reports\ReportAdmin) class for a better experience.
```
There are unlikely to be subclasses.
Also adding API link per https://docs.silverstripe.org/en/contributing/documentation/#links-to-api-documentation